Cargo.toml

```diff
@@ -14,34 +14,34 @@ exclude = ["doc/pastel.gif"]
 
 [dependencies]
 # library dependencies
-lazy_static = "1.3.0"
-rand = "0.7"
-atty = "0.2"
-output_vt100 = "0.1"
-nom = "5"
+lazy_static = "1.4.0"
+rand = "0.8.4"
+atty = "0.2.14"
+output_vt100 = "0.1.2"
+nom = "7.0.0"
```

Comment:
Why did you add minor versions here? This makes maintenance hard, IMO. The lock file already stores the compatible minor versions (which can be used with `cargo … --locked`)

Reply:
I just run cargo upgrade from cargo-edit, I can revert that

Reply:
Yes please.

Reply:
I updated as you suggested, is that correct now?